apps.yaml

```diff
@@ -201,12 +201,14 @@ FullProfAPP:
         description: |
             An PyQt5 based graphical user interface for the automated analysis of powder X-ray diffraction data based on the refinement engine FullProf.
         authors: Oier Arcelus, Juan Rodriguez-Carvajal, Nebil Ayape Katcho
-        external_url: https://code.ill.fr/rodriguez-carvajal/fullprof-app
+        external_url: https://www.ill.eu/sites/fullprof/php/downloads.html
         logo: https://code.ill.fr/rodriguez-carvajal/fullprof-app/-/raw/master/src/fpgui/icons/splash.png
         state: development
+        version: 1.0
     categories:
         - data-analysis
         - utilities
+    git_url: https://code.ill.fr/rodriguez-carvajal/fullprof-app
```

Comment:
this is not a github url, please remove it.

Reply:
The URL links to a GitLab repo. This was the link that was included originally in the last pull request I did, I've just moved it from 'metadata' to 'git_url'.

Reply:
This url is not working for me, so I cannot check if it is indeed a gitlab url. Please note that only github and gitalb url are allowed in the `git_url` field. If you want to link some other websites, you can use `external_url` field.

Reply:
These changes do not apply to the latest commit